Application Actions/Spotify/1.0/PlayPause.scpt

```diff
@@ -0,0 +1 @@
+tell application "Spotify" to playpause
```

Comment:
Can you please rename the file to `Play or Pause.scpt`? You then also need to update `default.json`